src/libs/tools/include/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 file (GLOB HDR_FILES
-	   *.hpp)
+	   *.hpp
+	   newmerge.h)
```

Comment:
kdbmerge.h can be directly in src/include (it will be an independent lib, unrelated to libtool)

Reply:
This was a really helpful hint into the right direction. Thank you!